src/libstore/build.cc

```diff
@@ -809,6 +809,9 @@ class DerivationGoal : public Goal
     /* Whether this is a fixed-output derivation. */
     bool fixedOutput;
 
+    /* Whether this is a content adressed derivation */
+    bool contentAddressed = false;
```

Comment:
Maybe not add this and use a temporary below in hopes that #3418 is merged soon?

Reply:
Mh right. I've removed it in favor of directly calling `parsedDrv->contentAddressed()`